calculator.js

```diff
@@ -1,5 +1,70 @@
-const exampleAdditionInput = {
+//check input is valid
+function number_validation(number) {
```

Comment:
Just for style, you should make it camelCase and not snake_case.  Also maybe `validNumber` makes a better function name.

```suggestion
function validNumber(number) {
```